README.md

```diff
@@ -26,14 +26,6 @@ cd argo
 bundle install
 ```
 
-Note that `bundle install` may complain if MySQL isn't installed.  You can either comment out the `mysql2` inclusion in `Gemfile` and come back to it later (you can develop using `sqlite3`), or you can install MySQL.
-
-Install foreman for local development (foreman is not supposed to be in the Gemfile, See this [wiki article](https://github.com/ddollar/foreman/wiki/Don't-Bundle-Foreman)):
-
-```bash
-gem install foreman
-```
```

Comment:
Newer `bin/dev` scripts (such as is in Argo) automatically install `foreman` if missing.